### ISSUE TITLE
Normalized Mutual Information (NMI) and Generalizability (G)

### ISSUE THREAD

**Issue by nigeljyng** [fferroni/DEC-Keras#1]
Are there any plans to implement NMI and G metrics as outlined in the paper?

Looking at the original Caffe version of DEC, NMI is defined using `sklearn.metrics.normalized_mutual_info_score`. https://github.com/piiswrong/dec/blob/master/dec/dec.py#L701

As for G, it should be a straightforward division of training loss and validation loss. 

I plan to use use this to play around with some clustering. If/when I do I'm happy to take a stab at it. Or if you have time to work on it before then that would be great too.

**Comment by fferroni** [comment]
No but feel free to open a pull request
